std/fmt.zig

```diff
@@ -20,17 +36,28 @@ pub fn format(
     comptime fmt: []const u8,
     args: ...,
 ) Errors!void {
+    if (args.len > 32) {
```

Comment:
Arbitrary choice. Can be increased but I think this is a suitable limit for now anyway. I'm unsure you can even print 32 arguments without hitting the default comptime evaluation limit now anyway.